src/compiler/reviews.md

```diff
@@ -27,11 +27,74 @@ delegate+` or `@bors delegate=username`. This will allow the PR author
 to approve the PR by issuing `@bors` commands like the ones above
 (but this privilege is limited to the single PR).
 
-## High priority issues
+## Rollups
 
-When merging high priority issues (`P-critical` and `P-high`) it's
-recommended to avoid rollups and bump a bit the priority of the PR in
-the homu queue by issuing `@bors r+ rollup=never p=1`.
+All reviewers are strongly encouraged to explicitly mark a PR as to whether or
+not it should be part of a [rollup] with one of the following:
+
+- `rollup=always`: These PRs are very unlikely to break tests or have performance
+  implications. Example scenarios:
+    - Changes are limited to documentation, comments, etc. that is highly
+      unlikely to fail a build.
+    - Changes are pure refactoring and cannot have performance implications.
+    - Your PR is not landing possibly-breaking or behavior altering changes.
+        - Feature stabilization without other changes is likely fine to
+          rollup, though.
+- `rollup=maybe`: This is the **default** if you do not specify a rollup
+  status. Use this if you don't have much confidence that it won't break
+  tests. This can be used if you aren't sure if it should be one of the other
+  categories.
```

Comment:
I assume rollupers likely add `rollup=maybe` PRs to rollup. So, I think it'd be nicer if we mentioned it and encouraged members to use `iffy` rather than `maybe` if there's any concerns?

Reply:
Hm, I don't really know.  Like, I imagine there are quite a few PRs that do not fit the category of "iffy" (that is, "risky"), but the reviewer doesn't have enough confidence to mark it "always".  Or, what is probably more common, is "I don't know what it should be", which I think is fine since I don't think we can expect everyone to be deeply familiar with the process and hazards of rollups.

@Manishearth any opinions?

Reply:
I basically think folks should never be explicitly marking things as rollup=maybe, it's the default, the reason you do it is to unmark something else from a diferent rollup level.